app/Http/Controllers/Api/V1/TeamDataAccessApplicationController.php

```diff
@@ -769,12 +769,19 @@ public function edit(EditDataAccessApplication $request, int $teamId, int $id):
                 'dar_application_id' => $id,
             ])->first();
 
+            $submissionStatusUpdate = $input['submission_status'] ?? null;
+
             $reviewId = null;
             if (isset($input['comment'])) {
-                $review = DataAccessApplicationReview::create([
-                    'application_id' => $id,
-                    'resolved' => true,
-                ]);
+                if ($submissionStatusUpdate === 'DRAFT') {
+                    $review = DataAccessApplicationReview::where('application_id', $id)->first();
+                }
+                if (!$review) {
```

Comment:
only when status is draft, otherwise is not defined?

Reply:
That's correct, the comment is added to an existing review if the status is being changed to "DRAFT", if there is an existing review.  Otherwise a new review is created.